foundationsync/indico_foundationsync.py

```diff
@@ -104,9 +112,9 @@ def _parse_room_data(self, raw_data, coordinates, room_id):
         data['capacity'] = int(raw_data['CAPACITY']) if raw_data['CAPACITY'] else None
         data['surface_area'] = int(raw_data['SURFACE']) if raw_data['SURFACE'] else None
         data['division'] = raw_data.get('DEPARTMENT')
-        data['telephone'] = raw_data.get('TELEPHONE')
-        data['key_location'] = raw_data.get('WHERE_IS_KEY')
-        data['comments'] = raw_data.get('COMMENTS')
+        data['telephone'] = raw_data.get('TELEPHONE') if raw_data.get('TELEPHONE') else ''
+        data['key_location'] = self._fix(str(raw_data.get('WHERE_IS_KEY'))) if raw_data.get('WHERE_IS_KEY') else ''
```

Comment:
This `str(...)` looks like it's gonna blow if there are non-ASCII chars. I'm not 100% sure since I don't know what the underlying cxOracle implementation is returning, but I'd just double check that it works ok with accented characters.

Reply:
AFAIK we configured the oracle client to return unicode objects somewhere near the top of the file, so `str()` seems likely to break in that case